client/src/app/+admin/admin.component.ts

```diff
@@ -1,12 +1,34 @@
 import { Component } from '@angular/core'
 import { UserRight } from '../../../../shared'
 import { AuthService } from '../core/auth/auth.service'
+import { ListOverflowItemComponent } from '@app/shared/misc/list-overflow.component'
+import { I18n } from '@ngx-translate/i18n-polyfill'
 
 @Component({
   templateUrl: './admin.component.html'
 })
 export class AdminComponent {
-  constructor (private auth: AuthService) {}
+  items: ListOverflowItemComponent[] = []
+
+  constructor (
+    private auth: AuthService,
+    private i18n: I18n
+  ) {
+    this.items = this.buildItemsList()
```

Comment:
Put this in `ngOnInit` so it only loads the items list when the component initializes (kind of lazy load). It's a good practice so the web browser does not initialize all components when it loads the bundle and the angular application (it's slow enough :) )